src/ConcreteOperations/isdisjoint.jl

```diff
@@ -744,12 +744,31 @@ function is_intersection_empty(hp::Union{Hyperplane{N}, Line2D{N}},
     return is_intersection_empty_helper_hyperplane(hp, X, witness)
 end
 
-# disambiguation
 function is_intersection_empty(hp1::Union{Hyperplane{N}, Line2D{N}},
                                hp2::Union{Hyperplane{N}, Line2D{N}},
                                witness::Bool=false
                               ) where {N<:Real}
-    return is_intersection_empty_helper_hyperplane(hp1, hp2, witness)
+    if isequivalent(hp1, hp2)
+        res = false
+        if witness
+            w = an_element(hp1)
+        end
```

Comment:
This case is needed because `intersection(hp1, hp2)` would return an `HPolyhedron` and then `an_element` would return `[Inf, Inf]`.